src/processors/data_processor.py

```
@@ -188,7 +188,9 @@ def _process_notifiable(self, lines: list[str], source_file: Path, metadata: dic
                 success=False, source_path=source_file, error="全数報告処理中にエラーが発生しました"
             )
 
-    def _process_sentinel(self, lines: list[str], source_file: Path, metadata: dict[str, Any]) -> NormalizationResult:
+    def _process_sentinel(  # noqa: PLR0912  # 性別セクション処理+medical_district特別処理+total検証の複合ロジックのため複雑
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR0912`)

Remove unused `noqa` directive

(RUF100)